packages/input/src/input.tsx

```diff
@@ -6,11 +6,28 @@ import {
   ThemingProps,
   useMultiStyleConfig,
   HTMLChakraProps,
+  PropsOf,
 } from "@chakra-ui/system"
 import { cx, __DEV__ } from "@chakra-ui/utils"
 import * as React from "react"
 
-interface InputOptions {
+interface NativeInputOptions {
+  /**
+   * The native HTML `size` attribute to be passed to the `input`
+   */
+  htmlSize?: number
+}
+
+interface NativeInputProps extends PropsOf<"input">, NativeInputOptions {}
+
+const NativeInput = React.forwardRef(
+  (props: NativeInputProps, ref: React.Ref<any>) => {
+    const { htmlSize, ...rest } = props
+    return <input size={htmlSize} ref={ref} {...rest} />
+  },
+)
+
```

Comment:
I think we can solve this without introducing an additional component. 
The intended behavior can be achieved by using the chakra factory function `<chakra.input />`

Reply:
Indeed that works. Definitely cleaner.